src/plugins/csvstorage/csvstorage.c

```diff
@@ -204,6 +205,9 @@ static int csvRead(KeySet *returned, Key *parentKey, char delim, short useHeader
 		offset = 0;
 		colCounter = 0;
 		char *lastIndex = "#0";
+		if(getColumnCount(lineBuffer, delim) != columns)
```

Comment:
Can you fix this and rebase this merge request? Then I can apply this before #311.